### PR TITLE
Rework manifesto

### PR DESCRIPTION
**Pull request by brikis98**
I left a bunch of comments on the manifest on the Google Doc. As requested, I've turned those comments/suggestions into a PR. I've ended up reworking the manifesto to:

1. More clearly focus on the central issue: that Terraform needs to remain under a truly open source license, and that without it, the community will fall apart. I think this is more important than the focus on whether Terraform is a language or not.
2. More clearly call out what we intend to do: namely, ensure Terraform ends up in a foundation.
3. (Minor) Switch to a narrower width + justified text, as I believe that's more pleasant for reading on larger screens. See screenshot below for how it looks now.

Screen Shot 2023-08-14 at 3 32 40 PM

This is a pretty significant change. We can choose to use all of it or cherry pick some parts or whatever else ya'll prefer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -160,10 +148,25 @@ <h1>The OpenTF Manifesto</h1>
...
+        
+        <p>
+          We ask HashiCorp to do the right thing by the community and to donate Terraform to this foundation,
+          rather than going forward with the BSL license change. That way, instead of fracturing the community, 
+          we end up with a single, impartial, reliable home for Terraform where the whole community can come together 
+          to keep building this amazing ecosystem.
+        </p>        
+
+        <p class="font-bold">
+          Our fallback plan: fork Terraform.
```
Comment (orbitz) [inline index.html:162]: I don't think we should mention this part.  We should just be asking to donate.

File: index.html [hunk 2]
```diff
@@ -160,10 +148,25 @@ <h1>The OpenTF Manifesto</h1>
...
+        </p>
+
+        <p>
+          If HashiCorp is unwilling to donate Terraform to the foundation, then we propose to fork Terraform, and
```
Comment (orbitz) [inline index.html:166]: I don't think we should have an ultimatum

File: index.html (hunk 1 above)
Comment (DicsyDel) [reply index.html:115]: Agreed.

File: index.html (hunk 2 above)
Comment (DicsyDel) [reply index.html:119]: +1

File: index.html (hunk 1 above)
Comment (brikis98) [reply index.html:114]: Then what are we pledging to do at the end?

To contribute to the foundation if HashiCorp donates to it? 

And if they choose not to? Or just ignore the manifesto?

Don't manifesto's typically promise action?

Comment (marcinwyszynski) [reply index.html:114]: To contribute to the foundation regardless of whether Hashi donates it or not.

**Review by marcinwyszynski — APPROVED** [review]
I like your version better than mine @brikis98

**Comment by SebastianStadil** [comment]
I like this new version better too.

File: index.html (hunk 1 above)
Comment (orbitz) [reply index.html:114]: IMO we write this with the assumption that HCP will donate it.

**Comment by brikis98** [comment]
While merging the latest from `main`, I hit a broken `<h2>` block. I think it was from this PR: https://github.com/opentffoundation/manifesto/pull/21/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R116-R117.

I tried to fix it up a bit in this PR.

File: index.html [hunk 3]
```diff
@@ -9,81 +9,85 @@
...
-          their cloud presence suddenly became a legal risk. Thousands of people,
-          often deep in love with Terraform, found themselves in a position
-          where their hard-earned skills are in danger of becoming irrelevant.
+          Overnight, tens of thousands of businesses, ranging from one-man shops
+          to the Fortune 500, woke up to a new reality where the underpinnings of
+          their infrastructure suddenly became a potential legal risk. The legal terms in
+          the BSL license are vague, and now every company, every vendor, and every 
+          developer who is using Terraform has to wonder whether what they are doing could
+          be construed as competitive with HashiCorp. And even if you might be in the clear now,
+          how can you build confidence that your usage won't violate the license terms in the future?
```
Comment (coryodaniel) [inline index.html:36]: This is important. Worth a bold/stress?

**Comment by ZIJ** [comment]
No major objections

Suggest we keep the central point of "Terraform is language, not a hostable backend product" though. It is strong, easy to pass on, easy to relate, nerdy enough and simple enough at the same time.

File: index.html [hunk 4]
```diff
@@ -9,81 +9,85 @@
...
 
         <p>
-          The main difference between HashiCorp products like Vault or Waypoint
-          that are no longer open source and Terraform is plain and simple. At
-          its heart, Terraform is not a product. It’s a language. Much like the
-          language it’s built on - Go. Imagine an alternate reality where the
-          corporate sponsor behind Go decides to relicense the compiler only to
-          allow writing software that does not negatively impact its bottom
-          line.
+          And this sort of change not only fractures the Terraform community, but it
```
Comment (coryodaniel) [inline index.html:53]:
```suggestion
          This sort of change not only fractures the Terraform community, but it
```

File: index.html [hunk 5]
```diff
@@ -9,81 +9,85 @@
...
-          decided you could only run it in production on top of their public
-          cloud?
-        </p>
-
-        <p>
-          There is a reason why the building blocks of the modern Internet are
-          handed over to impartial bodies who can ensure that they become solid
-          and predictable underpinnings of our industry. This situation is no
-          different. As a community, we can take responsibility for our path.
+          This is why the building blocks of the modern Internet, such as Linux and Kubernetes, are
```
[the remark this answers is not in the document]
Comment (brikis98) [reply index.html:63]: Thanks!

File: index.html [hunk 6]
```diff
@@ -9,81 +9,85 @@
...
-          and predictable underpinnings of our industry. This situation is no
-          different. As a community, we can take responsibility for our path.
+          This is why the building blocks of the modern Internet, such as Linux and Kubernetes, are 
+          handed over to impartial bodies (the Linux Foundaiton and the Cloud Native Computing 
+          Foundation, respectively) who can ensure that those building blocks can form solid and 
+          predictable underpinnings for our industry.
         </p>
 
+        <p class="font-bold">
+          Our goal: ensure Terraform is handed over to a foundation.
```
Comment (coryodaniel) [inline index.html:70]: Worth clarifying “open-source foundation”?

File: index.html [hunk 7]
```diff
@@ -96,10 +100,25 @@ <h1>The OpenTF Manifesto</h1>
...
         <p class="font-bold">
-          We ask HashiCorp to do right by the community and donate Terraform to
-          a reputable foundation where, together with the community and
-          resources that we pledge, we continue to build and maintain this
-          amazing ecosystem.
+          Our request of HashiCorp: donate Terraform to this foundation.
+        </p> 
+        
+        <p>
+          We ask HashiCorp to do the right thing by the community: donate Terraform to this foundation,
```
Comment (coryodaniel) [inline index.html:107]: L 103/107 restatement back-to-back briefly confused me and caused me to reread the section

File: index.html (hunk 3 above)
Comment (brikis98) [reply index.html:40]: Good thought. I added a powerful, bold header: https://github.com/opentffoundation/manifesto/pull/20/commits/910ae0e2a695b976981ea34577483186d0050742

"Our concern: the BSL license is a poison pill for Terraform."

File: index.html (hunk 2 above)
Comment (coryodaniel) [reply index.html:122]: I don’t think it should be aggressively phrased as an ultimatum but needs to be clear to the reader that this is not a whimpering request.

File: index.html (hunk 7 above)
Comment (brikis98) [reply index.html]: Slightly rephrased: https://github.com/opentffoundation/manifesto/pull/20/commits/124b16c35d9576871b42e3336bf967e8f57bd8fa. LMK if clearer.

File: index.html (hunk 6 above)
Comment (brikis98) [reply index.html]: Updated: https://github.com/opentffoundation/manifesto/pull/20/commits/124b16c35d9576871b42e3336bf967e8f57bd8fa

**Pull request merged**